care/facility/models/patient_icmr.py

```diff
@@ -154,6 +154,14 @@ def symptoms(self):
             if SYMPTOM_CHOICES[0][0] not in self.consultation.symptoms.choices.keys()
         ]
 
+    @property
+    def date_of_onset_of_symptoms(self):
+        return (
+            self.consultation.symptoms_onset_date.date()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Instance of 'ForeignKey' has no 'symptoms_onset_date' member](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5348154)